### PR TITLE
fix: Cannot read property 'update' of null

### PR DESCRIPTION
**Pull request by donnes**
Remove the necessity to use event `.update()` from Charts.js API

### CODE REVIEW

File: src/Chartjs.vue [hunk 1]
```diff
@@ -49,7 +45,7 @@ export default {
       this.$nextTick(() => {
         // this.chart.data.datasets = val.datasets
         // this.chart.data.labels = val.labels
-        this.chart.update()
+        this.mountChart()
       })
```
Comment (fundon) [inline src/Chartjs.vue:49]: @donnes Thanks

check first is ok?

``` js
if (this.chart) {
  this.chart.update()
}
```

Comment (donnes) [reply src/Chartjs.vue:49]: @fundon I thank you for the good work.
About the question, with this condition the error disappears, but the canvas doesn't draw

Comment (fundon) [reply src/Chartjs.vue:49]: Try to `console.log(val)`, Is `val` changed?

Comment (fundon) [reply src/Chartjs.vue:49]: I think we should keep the `chart` instance, don't create new instance every times. 
Or add a `force` option for that.

Comment (donnes) [reply src/Chartjs.vue:49]: @fundon Agree, I will try a best solution to this problem

**Pull request closed without merging**